packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -40,178 +39,204 @@ import { API_ERROR_CODES } from 'Constants/api-error-codes';
+                        ? CLAIMED_DOCUMENT_ERROR_MESSAGE
+                        : idv_update_response.error?.message ?? GENERIC_ERROR_MESSAGE;
+                setStatus({ error_msg: response_error });
+                setSubmitting(false);
+                return;
+            }
             setSubmitting(false);
-            return;
-        }
-        setSubmitting(false);
-        handleViewComplete();
-    };
-
-    return (
-        <Formik initialValues={{ ...initial_values }} validate={validateFields} onSubmit={submitHandler}>
-            {({ dirty, isSubmitting, isValid, values, status }) => (
-                <Form className='proof-of-identity__container proof-of-identity__container--reset'>
-                    {status?.error_message && (
-                        <div className='account-form__poi-confirm-example--generic-error-msg'>
-                            <HintBox
-                                icon='IcAlertDanger'
-                                message={
-                                    <Text as='p' size='xxxs'>
-                                        {status.error_message}
-                                    </Text>
-                                }
-                                is_danger
-                                className='hint-box-layout'
-                            />
-                        </div>
-                    )}
-                    <FormBody className='form-body' scroll_offset={isMobile() ? '180px' : '80px'}>
-                        <FormSubHeader title={localize('Identity verification')} />
-                        <IDVForm selected_country={selected_country} class_name='idv-layout' />
-
-                        <FormSubHeader title={localize('Details')} />
-                        <PersonalDetailsForm
-                            class_name='account-form__poi-confirm-example_container'
-                            is_rendered_for_idv
-                            editable_fields={values.confirmation_checkbox ? [] : changeable_fields}
-                            side_note={side_note_image}
-                            inline_note_text={
-                                <Localize
-                                    i18n_default_text='To avoid delays, enter your <0>name</0> and <0>date of birth</0> exactly as they appear on your identity document.'
-                                    components={[<strong key={0} />]}
+            handleViewComplete();
+        };
+
+        return (
+            <Formik initialValues={{ ...initial_values }} validate={validateFields} onSubmit={submitHandler}>
+                {({ dirty, isSubmitting, isValid, values, status }) => (
+                    <Form className='proof-of-identity__container proof-of-identity__container--reset'>
+                        {status?.error_message && (
+                            <div className='account-form__poi-confirm-example--generic-error-msg'>
+                                <HintBox
```

Comment:
Can we update the hintbox to inline message as it's deprecated

Reply:
yes. Previously we had no clear explanation, why it was deprecated. now I'll try to replace it.

Reply:
I think it would be better to make it in some of our refactoring cards